client/web/src/components/panels/Document.vue

```diff
@@ -149,7 +149,7 @@
 			flex: 1 1 100%;
 
 			.canvas {
-				background: var(--color-1-nearblack);
```

Comment:
Could you specifically apply the `background: #ffffff;` to the `svg` tag below, and restore `background: var(--color-1-nearblack);` on the `.canvas`?